website/docs/reference/project-collaboration-mode.md

```diff
@@ -27,7 +27,7 @@ Only admins and project members can submit change requests in a **protected** pr
 
 Private collaboration mode renders the project invisible to all **viewers** who are not project members. This means that **viewers** cannot see the project in the project list, nor can they access the project's features or locate the project anywhere within Unleash.
 
-**Editors** and **admins** can still see private projects.
+**Editors**, **admins**, and **custom root roles** can still see private projects.
```

Comment:
Maybe change this a bit? It feels weird to me when referring to users as "custom root roles". Editors and admins is fine, but "custom root roles" makes it sound like you're talking about the roles themselves and not the users.

```suggestion
**Editors**, **admins**, and users with **custom root roles** can still see private projects.
```

Reply:
Appreciate the oxford comma, though ;)